app/models/deletion.rb

```diff
@@ -30,7 +30,7 @@ def record_metadata
   end
 
   def expire_api_memcached
-    ["deps/v1/#{rubygem}", "info/#{rubygem}", "versions", "names"].each do |key|
+    ["deps/v1/#{rubygem}", "versions", "names"].each do |key|
```

Comment:
@segiddins 
> where is it that the cache gets set?

cache gets set on `GET` request for `info/foo`. Since I am explaining it now, it is most likely thats not what you meant by **fastly re-requests the page**. Can you please explain?